Test/TypeFamilyTH.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE TemplateHaskell       #-}
+{-# LANGUAGE TypeFamilies          #-}
+{-# LANGUAGE UndecidableInstances  #-}
+
+module Main where
+
+import           Opaleye.Internal.TypeFamilies
```

Comment:
Is it possible to use `Opaleye.TypeFamilies` instead, i.e. not use the `Internal` module?  That would be preferable.